markode/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.4.0",
     "@testing-library/user-event": "^7.2.1",
+    "node-modules": "^1.0.1",
```

Comment:
according to [documentation](https://www.npmjs.com/package/node-modules), this package is intended to be installed globally. are you sure you want to integrate it in the project?